app/helpers/data.py

```diff
@@ -83,20 +82,7 @@ def push_user_notification(user):
         """
```

Comment:
Can't we remove this function entirely as the description says that it is for pushing notifications using websockets?